pulp_smash/tests/platform/api_v2/simple_repo_test.py

```diff
@@ -0,0 +1,174 @@
+# coding=utf-8
+"""Test for basic repo creating functionality.
+"""
+from __future__ import unicode_literals
+
+from pulp_smash.resources.platform.api_v2.api import Repository, Task
+from pulp_smash.resources.platform.api_v2.api import ERROR_KEYS
+from unittest2 import TestCase
+
+
+class RepoCreateSuccessTestCase(TestCase):
+    """Tests for successfull repo creating functionality.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create repo on pulp server
+        """
+        cls.repo = Repository(id=cls.__name__)
+        cls.repo.create_repo()
+
+    def test_status_code(self):
+        """Test if Create repo returned 201."""
+        self.assertEqual(self.repo.last_response.status_code, 201)
+
+    def test_correct_id(self):
+        """Test if response contain correct repo id.
+        """
+        self.assertEqual(
+            self.repo.last_response.json()['id'],
+            self.__class__.__name__,
```

Comment:
Please avoid using names like this. You can access an object's class with `type(self)` instead of `self.__class__`. And the `__name__` reference can be dropped in favor of randomly generating an ID, which has benefits as described here https://github.com/PulpQE/pulp-smash/pull/20/files#r43018157